### PR TITLE
[FIX] purchase_sale_inter_company: commitment_date

### PR DESCRIPTION
**Pull request by chienandalu**
The field was renamed from `requested_date` to `commitment_date`

cc @Tecnativa TT26981

### CODE REVIEW

File: purchase_sale_inter_company/models/purchase_order.py [hunk 1]
```diff
@@ -115,6 +115,7 @@ def _prepare_sale_order_data(self, name, partner, dest_company,
             'partner_id': partner.id,
             'date_order': self.date_order,
             'auto_purchase_order_id': self.id,
+            'commitment_date': self.date_planned,
```
Comment (pedrobaeza) [inline purchase_sale_inter_company/models/purchase_order.py]: Isn't this change in later onchanges?

Comment (chienandalu) [reply purchase_sale_inter_company/models/purchase_order.py]: As long as I tested it doesn't change

Comment (pedrobaeza) [reply purchase_sale_inter_company/models/purchase_order.py]: Well, I prefer to assign it at the end as the previous one for being sure it doesn't change

Comment (chienandalu) [reply purchase_sale_inter_company/models/purchase_order.py]: Ok, changed

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Let's fast-track this for fixing the bug:

/ocabot merge patch

Is this needed for v13?

**Comment by chienandalu** [comment]
Yes, it seems to have the same issue

**Review by bealdav — APPROVED** [review]

**Pull request merged**